Gemfile.lock

```diff
@@ -108,14 +109,15 @@ GEM
     omniauth-oauth2 (1.4.0)
       oauth2 (~> 1.0)
       omniauth (~> 1.2)
-    pg (0.18.4)
-    pry (0.10.3)
+    pg (0.19.0)
+    pry (0.10.4)
       coderay (~> 1.1.0)
       method_source (~> 0.8.1)
       slop (~> 3.4)
     pry-rails (0.3.4)
       pry (>= 0.9.10)
-    puma (2.15.3)
+    public_suffix (2.0.4)
+    puma (3.6.2)
```

Comment:
The puma upgrade is a major version bump. Did you check out if any configuration or deployment changes are in order?